### PR TITLE
Update Querying API docs based on additional feedback on #405

### PR DESCRIPTION
**Pull request by dmontagu**
Address Samuel's feedback on #405.

I'll note that I am fairly confident the `Accept` header is not required, but the default value sent by requests and various other tools is or contains `*/*`; I have a PR open on the platform in order to send JSON when the Accept header contains this.

### CODE REVIEW

File: docs/guides/advanced/query_api.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ Here's an example of how to use these clients:
             # Retrieve data in arrow format, and load into a polars DataFrame
             # Note that JSON columns such as `attributes` will be returned as
             # JSON-serialized strings
-            df_from_arrow = pl.from_arrow(await client.query_arrow(sql=query))
+            df_from_arrow = pl.from_arrow(await client.query_arrow(sql=query))  # type: ignore
```
Comment (alexmojaki) [inline docs/guides/advanced/query_api.md]: `# type: ignore` in docs????

File: docs/guides/advanced/query_api.md [hunk 2]
```diff
@@ -197,22 +194,21 @@ else:
...
-  - **`max_timestamp`**: Similar to `min_timestamp`, but serves as an upper bound for filtering `start_timestamp` in the `records` table or `recorded_timestamp` in the `metrics` table. The same filtering can also be done manually within the query itself.
-  - **`limit`**: An optional parameter to limit the number of rows returned by the query. If not specified, **the default limit is 500**. The maximum allowed value is 10,000.
-  - **`row_oriented`**: Only affects JSON responses. If set to `true`, the JSON response will be row-oriented; otherwise, it will be column-oriented.
+    - **`sql`**: The SQL query to execute. This is the only required query parameter.
+    - **`min_timestamp`**: An optional ISO-format timestamp to filter records with `start_timestamp` greater than this value for the `records` table or `recorded_timestamp` greater than this value for the `metrics` table. The same filtering can also be done manually within the query itself.
```
Comment (alexmojaki) [inline docs/guides/advanced/query_api.md]: Does this now have a default that should be documented?

File: docs/guides/advanced/query_api.md [hunk 3]
```diff
@@ -197,22 +194,21 @@ else:
 
 ### Additional Configuration
 
-The Logfire API supports various query parameters and response formats to give you flexibility in how you retrieve your data:
+The Logfire API supports various response formats and query parameters to give you flexibility in how you retrieve your data:
 
 - **Response Format**: Use the `Accept` header to specify the response format. Supported values include:
```
Comment (samuelcolvin) [inline docs/guides/advanced/query_api.md]: I don't think multiple levels of bullet point work in mkdocs by default.

File: docs/guides/advanced/query_api.md [hunk 4]
```diff
@@ -166,10 +166,7 @@ base_url = 'https://logfire-api.pydantic.dev'
 read_token = '<your_read_token_here>'
 
 # Set the headers for authentication
-headers = {
-    'Authorization': f'Bearer {read_token}',
-    'Content-Type': 'application/json'
-}
+headers = {'Authorization': f'Bearer {read_token}'}
```
Comment (samuelcolvin) [inline docs/guides/advanced/query_api.md]: Are you sure we don't need the `Accept` header?

File: docs/guides/advanced/query_api.md (hunk 3 above)
Comment (alexmojaki) [reply docs/guides/advanced/query_api.md]: It seems to be working: https://4d3d712c.logfire-docs.pages.dev/guides/advanced/query_api/#additional-configuration

File: docs/guides/advanced/query-api.md [hunk 5]
```diff
@@ -79,7 +79,7 @@ Here's an example of how to use these clients:
             # Retrieve data in arrow format, and load into a polars DataFrame
             # Note that JSON columns such as `attributes` will be returned as
             # JSON-serialized strings
-            df_from_arrow = pl.from_arrow(await client.query_arrow(sql=query))
+            df_from_arrow = pl.from_arrow(await client.query_arrow(sql=query))  # type: ignore
```
Comment (alexmojaki) [inline docs/guides/advanced/query-api.md]: ?

File: docs/guides/advanced/query-api.md [hunk 6]
```diff
@@ -197,22 +194,21 @@ else:
 
 ### Additional Configuration
 
-The Logfire API supports various query parameters and response formats to give you flexibility in how you retrieve your data:
+The Logfire API supports various response formats and query parameters to give you flexibility in how you retrieve your data:
```
Comment (alexmojaki) [inline docs/guides/advanced/query-api.md:197]: why were these swapped lol

Comment (dmontagu) [reply docs/guides/advanced/query-api.md:197]: to match the order the bullets occur below

**Pull request merged**